### PR TITLE
Add Solidly V3 to dex.trades

### PR DESCRIPTION
**Pull request by ibilbobaggins**
Now it should be available in dex.trades

### Contribution type
Please check the type of contribution this pull request is for:

- [ ] New spell(s)
- [x] Adding to existing spell lineage
- [ ] Bug fix

**Note:** You can safely discard any section below which doesn't apply based on selection above

---

### For new spell(s)
If you are building new spell(s), please provide the following information:

- **Spell name(s):** *schema.table_name*
- **Description:** [Detailed description of the new spell(s) and their purpose]
- **Who are the new spell(s) for?** [Internal team or general community]
- **How will the new spell(s) be used downstream?** [Description of downstream usage]
- **Implementation details:** [Information on how the spell(s) are implemented]
- **Test instructions:** [How to test the new spell(s)]
- **Related issue(s):** [Link to related issues, if any]

---

### For adding to existing spell lineage
If you are adding to an existing spell lineage, please provide the following information:

- **Description:** [Description of the changes made]

---

### For bug fixes
If you are fixing a bug, please provide the following information:

- **Description:** [Description of the bug fix]
- **Steps to reproduce:** [How to reproduce the bug]
- **Implementation details:** [Information on how the bug was fixed]
- **Test instructions:** [How to test the fix]
- **Related issue(s):** [Link to related issues, if any]

---

### Additional information
Please provide any additional information that might help us review your pull request:

- [Any additional information]

---

Thank you for your contribution!

### CODE REVIEW

**Comment by ibilbobaggins** [comment]
I have read the CLA Document and I hereby sign the CLA

recheck

File: models/_sector/dex/trades/ethereum/platforms/solidly_v3_ethereum_base_trades.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
+    )
+}}
+
+{{
+    uniswap_compatible_v3_trades(
+        blockchain = 'ethereum',
+        project = 'solidly',
+        version = '3',
+        Pair_evt_Swap = source('solidly_ethereum', 'SolidlyV3Pool_evt_Swap'),
```
Comment (ibilbobaggins) [inline models/_sector/dex/trades/ethereum/platforms/solidly_v3_ethereum_base_trades.sql:18]: Ideally to be consistent I would like to use solidly_v3_ethereum.SolidlyV3Pool_evt_Swap here,
But contract is not accepted as solidly_ethereum contract already exists.

**Comment by ibilbobaggins** [comment]
recheck

File: seeds/_sector/dex/_schema.yml [hunk 2]
```diff
@@ -2641,6 +2641,67 @@ seeds:
...
+        tx_hash: varbinary
+        evt_index: uint256
+        block_number: uint256
+        token_bought_address: varbinary
+        token_sold_address: varbinary
+        token_bought_amount_raw: uint256
+        token_sold_amount_raw: uint256
+        block_date: timestamp
+
+
```
Comment (Hosuke) [inline seeds/_sector/dex/_schema.yml:2689]:
```suggestion

```

File: seeds/_sector/dex/_schema.yml [hunk 3]
```diff
@@ -2641,6 +2641,65 @@ seeds:
...
+        tx_hash: varbinary
+        evt_index: uint256
+        block_number: uint256
+        token_bought_address: varbinary
+        token_sold_address: varbinary
+        token_bought_amount_raw: uint256
+        token_sold_amount_raw: uint256
+        block_date: timestamp
+  - name: solidly_v3_fantom_base_trades_seed
+    config:
```
Comment (Hosuke) [inline seeds/_sector/dex/_schema.yml:2689]:
```suggestion
        block_date: timestamp

  - name: solidly_v3_fantom_base_trades_seed
    config:
```

**Review by Hosuke — COMMENTED** [review]
LGTM.✅
Thank you @ibilbobaggins 

[Sample solidly_v3_arbitrum_base_trades](https://dune.com/queries/3817714)
[Sample solidly_v3_ethereum_base_trades](https://dune.com/queries/3817716)
[Sample solidly_v3_fantom_base_trades](https://dune.com/queries/3817712)

**Review by Hosuke — APPROVED** [review]

**Comment by ibilbobaggins** [comment]
@Hosuke Seems ready to merge I think.

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**